### ISSUE TITLE
error when installing SABnzbdplus via sherpa

### ISSUE THREAD

**Issue by RAG-Dev** [OneCDOnly/sherpa#30]
Hi
I'm getting the following error when attempting to install SABnzbdplus on a QNAP TVS-682T. Entware has installed ok via sherpa.

sherpa.sh (200108)

[ done ] Internet is accessible
[ done ] downloaded file (SABnzbdplus_190928.qpkg)
[ done ] updated Entware package list
[ done ] no IPKGs are required
[ fail ] Download & install PIP modules failed [1]

[ fail ] 'SABnzbdplus' install failed! [19]

As requested I've attached the sherpa.debug.log
[sherpa.debug.log](https://github.com/OneCDOnly/sherpa/files/4172925/sherpa.debug.log)

Any assistance would be greatly appreciated.

**Comment by OneCDOnly** [comment]
Hi @RAG-Dev , thanks for posting your log. :+1: 

Let's start by reinstalling Entware:
```
./sherpa.sh Entware
```

**Comment by RAG-Dev** [comment]
Hi thank you for the advice. I have tried the reinstall of Entware which was successful and then attempted another reinstall of SABnzbdplus, however, received the same on screen error. Updated debug log attached (i removed the previous one so should only contain most recent reinstalls).
[Entware then SAB reinstall - sherpa.debug.log](https://github.com/OneCDOnly/sherpa/files/4173868/Entware.then.SAB.reinstall.-.sherpa.debug.log)

**Comment by RAG-Dev** [comment]
Hi After I tried the above I went through what I had  done. I was having trouble so saw an earlier issue that required Entwre1.02 which I downloaded and installed manually through the QNAP App Centre. I have just stopped and uninstalled (removed) it through the app centre as well as I stopped Mono & Sonarr 3 (installed earlier today from EU QNAP store). Then reinstalled SAB with ./sherpa.sh SABnzbdplus which downloaded 101 required packages (86M) and also reinstalled Entware 1.02. All green for the on screen install status. Thank  you for your assistance.
